styles.css

```diff
@@ -144,32 +149,38 @@ a {
   font-family: 'Ballet';
 } */
 
+h2 {
+  padding-top: 16pt; /* Changed */
```

Comment:
can we use `px` or `rem` instead of `pt`? just want to keep consistent w/ the rest of the css.